api/nodes/serializers.py

```diff
@@ -110,6 +118,7 @@ def get_pointers_count(self, obj):
         return len(obj.nodes_pointer)
 
     def create(self, validated_data):
+        validated_data.pop('type')
```

Comment:
Actually, it would be even more ideal if we didn't have to worry about calling `super`, and `validated_data` never had `id` and `type` at this point.